rotkehlchen/chain/ethereum/modules/nfts.py

```diff
@@ -312,6 +314,19 @@ def delete_price_for_nft(self, asset: Asset) -> bool:
 
         return True
 
+    def _filter_ignored_nfts(self, nfts_data: Dict[ChecksumEvmAddress, List[NFT]]) -> Dict[ChecksumEvmAddress, List[NFT]]:  # noqa: E501
+        """Remove ignored NFTs from NFTs data."""
+        with self.db.conn.read_ctx() as cursor:
+            ignored_nfts = set(self.db.get_ignored_assets(cursor=cursor, only_nfts=True))
```

Comment:
why do you need to turn it into a set?

Reply:
To have an O(1) when checking if any NFT is present in the ignored nfts

Reply:
hmm okay. Can you add a small comment for this?